src/pendulum/duration.py

```diff
@@ -238,14 +238,22 @@ def in_minutes(self) -> int:
     def in_seconds(self) -> int:
         return int(self.total_seconds())
 
-    def in_words(self, locale: str | None = None, separator: str = " ") -> str:
+    def in_words(
+        self,
+        locale: str | None = None,
+        separator: str = " ",
+        seconds_n_decimal: int = 2,
```

Comment:
```suggestion
```
This is a separate feature to the PR, please raise a separate PR if you weant that